.gitignore

```diff
@@ -11,3 +11,4 @@ public/system
 log/*.log.*
 .idea
 .bundle/*
+.byebug_history
```

Comment:
:bulb: If `pry-byebug` is used instead, I don't think you'd have to add this line to the ignore file. I double checked my other projects and didn't see that I had added a line this to my `.gitignore` files so I think this is correct.